src/store/index.ts

```diff
@@ -68,11 +66,11 @@ const reducers = combineReducers({
 })
 
 export type AppReduxState = CombinedState<{
-  [PROVIDER_REDUCER_ID]?: ProviderRecord
+  [PROVIDER_REDUCER_ID]?: ProviderState
```

Comment:
all good, just a question: do you know why this is optional (`?`)? it's always present in the redux store